### PR TITLE
ENH: Add ITKShape as a remote module

### PR DESCRIPTION
**Pull request by tbirdso**
Adds [`ITKShape`](https://github.com/slicersalt/itkshape) as a remote module to be built with ITK as requested by @thewtex .

Note that pending [Issue 22](https://github.com/slicersalt/ITKShape/issues/22) the module will build under the CMake project name `Mesh3DProcrustesAlignFilter` (its original name).

## PR Checklist
- [x] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [x] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)

Closes https://github.com/slicersalt/ITKShape/issues/22.

### CODE REVIEW

File: Modules/Remote/Shape.remote.cmake [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#-- # Grading Level Criteria Report
+#-- EVALUATION DATE: 2021-09-13
+#-- EVALUATORS: [<<NO EVALUATOR>>,<<NO EVALUATOR>>]
```
Comment (dzenanz) [inline Modules/Remote/Shape.remote.cmake:3]: Add your name here 😄

**Review by dzenanz — COMMENTED** [review]
Looks good.

**Comment by dzenanz** [comment]
Please update to the latest SHA, then we can merge it.

**Review by dzenanz — APPROVED** [review]

**Comment by dzenanz** [comment]
Merge when CI comes back green.

**Comment by tbirdso** [comment]
@dzenanz I don't have write access to ITK, could you please merge when tests complete or add me to the repo? Thank you!

**Review by thewtex — APPROVED** [review]
Thank you @tbirdso !

**Pull request merged**